sqlglot/expressions.py

```diff
@@ -4282,6 +4282,13 @@ class AlterTable(Expression):
     }
 
 
+class AlterView(Expression):
+    arg_types = {
+        "this": True,
+        "expression": False,
+    }
```

Comment:
Not sure if this is how we wanna implement this. For `CREATE ...` statements we have a single `Create` node which contains a `kind` argument that stores `TABLE`, `VIEW` etc. Maybe we want to do the same for `Alter` statements.

Reply:
Thanks for reviewing!
Yes that makes sense. Let me add that.